habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -283,17 +299,23 @@ def __init__(
             goal_sensor_keys = {
                 IntegratedPointGoalGPSAndCompassSensor.cls_uuid,
                 ObjectGoalSensor.cls_uuid,
+                ObjectCategorySensor.cls_uuid,
```

Comment:
Same comment as before. We should not hard code these but read their uuids from a config field. This is important when we want to run experiments with different sensor inputs.

Reply:
Currently, the uuids are actually read from a config field (gym.obs_keys) and I control the sensors that are passed. Any new 1d sensors will also get automatically appended to fuse_keys. 

But I will remove the ObjectEmbeddingSensor from here as you suggested.